lib/active_merchant/billing/credit_card.rb

```diff
@@ -245,7 +245,7 @@ def last_name?
       #
       # @return [String] the full name of the card holder
       def name
-        [first_name, last_name].compact.join(' ')
+        [first_name, last_name].compact.reject{ |n| n == "" }.join(' ')
```

Comment:
> Am I missing an edge case that negates the clarity of logic like this
Well, not that I know of. Honestly, I was wondering if there was some edge case I wasn't aware of so seemed safer to just use the existing array based approach. Using `strip` certainly seems clearer...